server/models/Availability.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const availabilitySchema = new mongoose.Schema({
+  availabilityDays: {
+    type: Array
+  },
+  availability: [{
+    day: {
```

Comment:
You can remove this file since we no longer are going to be focusing on availability in this pull request 👍🏻. We'll focus more on it when the time comes.